assets/externalized/game.json

```diff
@@ -79,6 +79,9 @@
     //  vanilla_setting: false
     "gui_extras": true,
 
+    //  Enables extra attachments: boosters and break lights on leggings, goggles on helmets
```

Comment:
Slightly OT, but I think we should move the comments to a separate file. Although there are some libraries that support comments in JSON files,  they are not part of the standard.

Reply:
that's exactly the reason we use one that does. This way is much more helpful for the user.

Reply:
We use a different library (`serde_json`) for the launcher. It doesn't look like it supports comments (Note: not tested, only skimmed the manual). If we want to include a `game.json` editor, it will become messy.

Reply:
It seems silly to me to write an editor, when we can just launch some platform default (notepade, xdg-open and ?? on mac).

Reply:
we have to move game.json first any way or it may get installed in a non-writable place. Uff, I can see a snowball nightmare ...